controller/scripts/entrypoint.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-exec python3.8 /driver/controller/controller_server/csi_controller_server.py $@
+exec python3.8 /driver/controller/controller_server/main.py $@
```

Comment:
Why not `__main__`?

Reply:
you mean add [`-m`](https://docs.python.org/3.8/library/__main__.html)?
couldn't it change some import behavior? it sounds to me like a separate ticket

Reply:
Yes. The CSI controller is a module with a "main" file. 
What do you mean by import behavior? No file should import the main file.

Reply:
I mean in general, `-m` has a general effect on imports in the project (e.g. [relative imports](https://stackoverflow.com/a/22250157)).
and I don't think this PR should affect the global scope of the project

Reply:
Next time :)